spec/features/users/authentication_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Authentication' do
+  def log_in(user)
```

Comment:
A minor point, but I'd move this to the bottom of the spec file (and I'm not sure if it should even be included in the `RSpec.feature` block, but taking it out might also break the test). I think it's better if the specs are read first and then you scroll down to view any abstracted methods.